py3status/module.py

```diff
@@ -350,6 +350,35 @@ def set_module_options(self, module):
 
             self.i3s_module_options['separator_block_width'] = sep_block_width
 
+        for name in ('background', 'border'):
+            if name in mod_config:
+                param = mod_config[name]
+                if not isinstance(param, basestring):
+                    raise TypeError(
+                        'Invalid attribute `{}`, '
+                        'should be an int. Got `{}`'
+                        .format(name, param)
+                    )
+
+                self.py3_module_options['block'][name] = (
+                    Py3(self)._get_color(param)
+                )
+
+                if name != 'border':
+                    continue
+                borders = ('top', 'bottom', 'left', 'right')
```

Comment:
I'd just prefer the real list than building them straight after, but not a biggy.

Why tuples not lists?

Reply:
please answer the question.

If you don't give me what I need how can I do a review?   Also please try to review your comments I find some very hard to follow.  Other times you can be clear so you have the skill if you are bothered, please be.

Reply:
I'm serious. Is it not faster than the list? Immutable. I'm using it for loops so I don't need to manipulate the list.. That's what I meant by not needing to touch the tuples.

Reply:
Maybe it does not matter because it does not apply to the line 359, but the next line down... Should be used with tuples, not list.

Reply:
please use a list it is clearer and no real other difference here.  If you want to save cycles there are better ways - readability wins over pretty much anything else.

Reply:
Okay, I'll switch to list right now.